src/routes/legal/privacy-policy.md

```diff
@@ -31,8 +33,13 @@ There is no reason for anyone outside of Project Segfault to see your data. In f
 
 ## Data may be cleared at any point upon request, but...
 
+There are times where we are unable to delete "your" data because we sometimes we do not have a concept of "you". If you don't create an account on the services you use, we have no way to know what data is tied to "you" as most logs are anonymized.
```

Comment:
imo using `when` instead of `where` is better, but you can leave it as is since you are a native speaker and i am not :)

Reply:
yep